public/docs/ts/latest/tutorial/_data.json

```diff
@@ -31,8 +31,8 @@
     "nextable": true
   },
   "toh-pt6": {
-    "title": "Http",
-    "intro": "We convert our service and components to use Http",
+    "title": "HTTP",
```

Comment:
The chapter title refers to HTTP (the protocol); we are not trying name a library or module (at this level).

HTTP is achieved via different means under TS and Dart (HttpModule vs. http library).